### PR TITLE
Splash #62

### PR DESCRIPTION
**Pull request by orjwan98**
for details: see commit history. 
references issue #62

### CODE REVIEW

File: client/src/components/SplashPage/splashpage.style.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import styled from 'styled-components';
+
+
+
+export const FatherContainer = styled.div`
+text-align: center;
```
Comment (jema28) [inline client/src/components/SplashPage/splashpage.style.js:6]: Please make sure all these indents are right.

**Review by jema28 — CHANGES_REQUESTED** [review]
Looking awesome but there are a lot of eslint issues. Please fix these 🍐 

Screenshot 2019-03-26 at 14 43 15

**Review by Hadeel-Salamin — APPROVED** [review]

**Review by amanshawar — APPROVED** [review]

File: client/src/components/SplashPage/index.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+};
+const Splash = props => (
+  <React.Fragment>
+    <FatherContainer>
+      <Title />
+      <AppDescription>
+        Ecommit makes it easy for you to trade your preloved items online.
+        Simply upload images of your items & allow our app to generate the data
+        needed to trade on.
+        <Span>Start by uploading a photo of your item</Span>
```
Comment (jema28) [inline client/src/components/SplashPage/index.js]: There's a full stop missing here.

**Comment by jema28** [comment]
Going to merge but the indents need to be fixed in a future PR.

**Review by jema28 — APPROVED** [review]

**Pull request merged**